client/my-sites/checklist/wpcom-checklist/checklist-banner/index.jsx

```diff
@@ -36,15 +33,15 @@ export class ChecklistBanner extends Component {
 
 	handleClose = () => {
 		const { siteId } = this.props;
-		const sitesNeverShowBanner = store.get( storeKeyForNeverShow ) || {};
-		sitesNeverShowBanner[ `${ siteId }` ] = true;
-		store.set( storeKeyForNeverShow, sitesNeverShowBanner );
+		setNeverShowBannerStatus( siteId, true );
 
 		this.setState( { closed: true } );
 
-		this.props.track( 'calypso_checklist_banner_close', {
-			site_id: siteId,
-		} );
+		if ( this.props.track ) {
+			this.props.track( 'calypso_checklist_banner_close', {
```

Comment:
Where do we (or don't) pass the `track` prop? I can't find it. `<ChecklistComponent>` doesn't have this prop when being rendered in `WpcomChecklistComponent`.

Usually we do analytics by dispatching actions from `state/analytics/actions`.

Reply:
I can't find any usage of it anywhere; must be old code. I remember I added this check because it kept failing; the error must have been there for a while. I'll remove it. Done in #28678.